### PR TITLE
chore(docs): Update litmus demo docs and fix indentation

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav <udit.gaurav@mayadata.io>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [x] Signed the commit for [DCO](https://github.com/litmuschaos/litmus-docs/blob/master/CONTRIBUTING.md#sign-your-work) check to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: docs/litmus-demo.md [hunk 1]
```diff
@@ -42,7 +43,89 @@ To get started with any of the above platforms we will follow the following step
   cd litmus-demo
   ```
 
-- Install the demo environment using one of the platforms with start argument
+### Setup Docker Container
```
Comment (ksatchit) [inline docs/litmus-demo.md]: Can we introduce the two ways of running demo? w/ docker & w/o it. Also, we can specify dockerized execution is supported for which all platforms

File: docs/getstarted.md [hunk 2]
```diff
@@ -119,7 +119,11 @@ Expected output:
 **NOTE**:
 
 - In this guide, we shall describe the steps to inject pod-delete chaos on an nginx application already deployed in the
-nginx namespace. It is a mandatory requirement to ensure that the chaos custom resources (chaosexperiment and chaosengine)
+nginx namespace. If you don't have this setup you can easily create one by running these two commands:
+  - Create nginx namespace `kubectl create ns nginx`.
+  - Create nginx deployment in nginx namespace `kubectl create deployment nginx --image nginx -n nginx`.
+
+- Now it is a mandatory requirement to ensure that the chaos custom resources (chaosexperiment and chaosengine)
```
Comment (ksatchit) [inline docs/getstarted.md]: We have both admin and ns mode of operation. So we can remove this para I suppose.

File: docs/litmus-demo.md (hunk 1 above)
Comment (uditgaurav) [reply docs/litmus-demo.md]: ok yes

File: docs/getstarted.md (hunk 2 above)
Comment (uditgaurav) [reply docs/getstarted.md]: Have removed!

**Review by ksatchit — APPROVED** [review]

**Pull request merged**